src/main/java/stream/flarebot/flarebot/commands/moderation/mod/MuteCommand.java

```diff
@@ -0,0 +1,92 @@
+package stream.flarebot.flarebot.commands.moderation.mod;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import net.dv8tion.jda.core.requests.RestAction;
+import org.joda.time.Period;
+import org.joda.time.format.PeriodFormatter;
+import org.joda.time.format.PeriodFormatterBuilder;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.mod.Punishment;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.scheduler.RestActionTask;
+import stream.flarebot.flarebot.util.GeneralUtils;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.awt.Color;
+
+public class MuteCommand implements Command {
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
+        if(args.length == 0){
+            MessageUtils.getUsage(this, channel, sender).queue();
+        } else {
+            User user = GeneralUtils.getUser(args[0], guild.getGuildId());
+            if(user == null){
+                MessageUtils.sendErrorMessage("Invalid user!!", channel);
+                return;
+            }
+            // NOT FOR v4
```

Comment:
How come?

Reply:
We are not doing mute stuff because it needs to be redone